src/components/aeIdentityMain/aeIdentityMain.js

```diff
@@ -0,0 +1,64 @@
+import aeIdentityAvatar from './../aeIdentityAvatar/aeIdentityAvatar.vue'
+import helperMixin from './../../mixins/helper'
+import BN from 'bn.js'
+
+/**
+ * Displays an Identity with an avatar blockie, the address and an amount of ether
+ */
+export default {
+  name: 'ae-identity-main',
+  components: { aeIdentityAvatar },
+  props: {
+    /**
+    * An object representing a identity. Must have a adress string a tokenBalance and a balance BigNumber (bn.js)
+    */
+    identity: {
+      type: Object,
+      default: () => ({
+        address: '0x0',
+        tokenBalance: new BN('0', 10),
+        balance: new BN('0', 10)
+      })
+    },
+    /**
+    * Is this the full size identity card or the collapsed one for displaying at the bottom of the screen?
```

Comment:
The collapsed identity displayed not only at the bottom of the screen but in identities list, probably in the address book and so.

Reply:
A question comment :D I'll remove that